front/src/components/login.js

```diff
@@ -18,21 +20,30 @@ export default class Login extends Component {
     }
 
     login(event) {
-        alert(this.state.email + "\n"
-            + this.state.password
-        );
-
         event.preventDefault();
+        AuthenticationService
+            .executeAuthenticationService(this.state.email, this.state.password)
+            .then(() => {
+                AuthenticationService.registerSuccessfulLogin(this.state.email, this.state.password);
+                this.props.history.push("/");
+            })
+            .catch(() => {
+                this.setState({error: true});
+                alert("Something went wrong")
```

Comment:
it would be good to tell specifically what went wrong (invalid username or password when response code is 401). Mybe not in this place but in `executeAuthenticationService`